go/vt/vtgate/planbuilder/operators/union.go

```diff
@@ -82,7 +81,7 @@ func (u *Union) AddPredicate(ctx *plancontext.PlanningContext, expr sqlparser.Ex
 	for i, selectExpr := range sel.SelectExprs {
 		ae, ok := selectExpr.(*sqlparser.AliasedExpr)
 		if !ok {
-			return nil, vterrors.Errorf(vtrpcpb.Code_UNIMPLEMENTED, "can't push predicates on UNION where the first SELECT contains star or next")
+			return nil, vterrors.VT12001("cannot push predicates on UNION where the first SELECT contains * or NEXT")
```

Comment:
Review all the `cannot` messages in this file to see whether they read well with `unsupported`.

Reply:
resolved in [f2351bb](https://github.com/vitessio/vitess/pull/10738/commits/f2351bba65f347a8e5ce72ff87757ccaf20115af)